scripts/setup-rp.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -exu
+
+# See https://developer.lightbend.com/docs/lightbend-orchestration/current/setup/cli-installation.html#install-the-cli for instructions for your platform
+
+wget -qO - https://downloads.lightbend.com/rp/keys/bintray-debian | \
+    sudo apt-key add - && \
```

Comment:
should we maybe stop using reactive-app? Another PR tho

Reply:
Yeah I don't think it adds much in this case - I didn't realize (until I prepared this PR) that `sbt 'deploy minikube'` actually needs the rp tool to be installed. Added #12